go/billing/models.py

```diff
@@ -176,6 +176,14 @@ class Meta:
         max_digits=10, decimal_places=3, default=Decimal('0.0'),
         help_text=_("The base cost per session in cents."))
 
+    session_unit_time = models.DecimalField(
+        max_digits=10, decimal_places=3, default=Decimal('20.0'),
+        help_text=_("The time of one billed unit (in seconds)."))
```

Comment:
I was under the impression we were going to keep this static. I've got a feeling we are going to need to assume a static unit time when we creating session length transactions from end-of-session messages, otherwise things could get pretty complicated. Maybe it isn't as complicated as I'm imagining though. @rudigiesler @hodgestar thoughts?